tests/storage/storage.go

```diff
@@ -1185,6 +1185,29 @@ var _ = SIGDescribe("Storage", func() {
 				}, 10*time.Second)).To(Succeed())
 			})
 
+			Context("With a USB device", func() {
```

Comment:
Would you mind adding a migration tests to ensure that migration works with this disk type too without issues? There may be a migration test already where we just attach a bunch of different disks, where you may be able to just add another disk.

Reply:
disk hotplug operations may need a test as well, what do you think?

Reply:
sure thing, sounds sensible.

Reply:
@rmohr right now when we add hotplug volumes we use the SCSI bus, does it make sense to extend that and support the USB bus as well? or support all bus types?

Right now hotplugging a USB disk is not possible.

Reply:
No, we are good then. If it is blocked similarly to virto-blk, then we are good. :+1:

Reply:
> If it is blocked similarly to virto-blk

What do you mean?

I've added a migration test btw.

Reply:
> What do you mean?
> 
> I've added a migration test btw.

 I just meant that when it works like `virtio` bus (virtio-blk), that disks cant' be hotplugged or unplugged (and not accidentally allowed somehow with the code chagnes),  we don't have to consider it. Should be ok.